### ISSUE TITLE
v4.7.2.1:  Cannot perform any backups (auto or manual)

### ISSUE THREAD

**Issue by pajohnson1066** [Drive4ik/simple-tab-groups#959]
**Describe the bug**
When trying to perform a backup of any type (auto scheduled or manual), a "Null" message appears.  The backup .json file is not created.
Auto backup:  The directory has been created (STG-backups-FF-102.0), but no backup files are created (even when setting the backup frequency at 1 minute).
Manual backup:  No backup file is created in any drive or folder location.

This stopped working correctly on 15 June 2022 (the last auto backup I have is "auto-stg-backup-day-of-month-14@drive4ik".json" and that is 4,700k in size.  I have had bigger files sizes in previous backup files.

Luckily, I have backed up the tabs to STG Bookmaks, so I can recover from the 14th June, but not anything since that date.

**To Reproduce**
1. Go to 'Setting icon in STG, bottom of sidebar'
2. Click on 'Backup your Simple Tab Groups' tab'
3. Click on 'Create backup'
4. Select folder location, filename is already created. Hit enter. 
5. Error box appears with the single word Null inside it (bottom right of screen).  No .json file created.
![image](https://user-images.githubusercontent.com/108526671/176886966-980e3a4c-53e9-4f96-9df5-60f3f2479c5e.png)

**Expected behavior**
A .json file is created in the folder/location selected. 

**Screenshots**

**Desktop (please complete the following information):**
 - Windows 10 Pro, Version 10.0.19044 Build 19044
 - Firefox version: 102.0 (64-bit)
 - Simple Tab Groups version: v4.7.2.1

**Additional context**
I have already re-installed the add on into FireFox, same behaviour since FF v101.0.1.

**Comment by nekohayo** [comment]
For what it's worth, I've upgraded my Firefox version from 101 to 102 on three computers to test this today, and I was **not** able to reproduce the issue on any of them. Automatic and manual backups all seem to work here (note that I'm using the default setting where the backups rotate based on the day of the month, reusing the same filenames). I'm running Fedora Linux, btw.

**Comment by nekohayo** [comment]
That's weird, because I had upgraded from every previous Firefox version before, so I did go through 100 -> 101 -> 102 here too, and I did not encounter the issue. In any case though, I would recommend that you please close the ticket/issue here (by using the web interface). Also you may want to edit your latest comment to remove all the data that was in your email signature and from your top-posted quoted email (and as a general rule, I would advise against top-posting quote replies via email for this very reason :)

**Comment by pajohnson1066** [comment]
It was a strange one, because the auto backup feature stopped working on 14th June, because that was the last day I had an auto backup file available to restore.  Now that I cleaned out the registry and re-installed FF, the plug in now works correctly, and performs auto backups as scheduled and also when I perform a manual backup, that works also.
Noted on the email reply, so I deleted that personal info from the posting above.  Thanks for that.
